### ISSUE TITLE
Why is the Pass extension not available on Android

### ISSUE THREAD

**Issue by I-I-IT** [ProtonMail/WebClients#414]
> We are happy to answer your questions about the code or discuss technical ideas.

I sideloaded the Proton Pass extension to Mull (Firefox) on Android, and it works great, enanching the experience, and passkey compatibility. However, Proton doesn't officially support this method. Why not?

**Comment by hisirdox** [comment]
They already have a proton pass app.

**Comment by I-I-IT** [comment]
> They already have a proton pass app.

I know, but it often doesn't work for website passkeys and detect les often website password fields.

**Comment by ManDDDDD** [comment]
> I know, but it often doesn't work for website passkeys and detect les often website password fields.

Have you set Proton Pass as your Autofill-Service in the device settings?

**Comment by I-I-IT** [comment]
> Have you set Proton Pass as your Autofill-Service in the device settings?

Yes, but the extension is just so much convenient. Especially if you use Graphene OS.
